src/devices/shinasystem.ts

```diff
@@ -1197,4 +1197,69 @@ export const definitions: DefinitionWithExtend[] = [
             }),
         ],
     },
+    {
+        zigbeeModel: ["HQM-300Z"],
+        model: "HQM-300Z",
+        vendor: "ShinaSystem",
+        ota: true,
+        description: "SiHAS Zigbee Wireless Round Thermostat",
+        extend: [
```

Comment:
Does it make sense to expose this device as a thermostat? https://github.com/Koenkk/zigbee-herdsman-converters/blob/db1d6f2ee92b1c05ea24b01745819e785e8c3095/src/lib/modernExtend.ts#L3173